react-ui/src/pages/analyzeVotes/AnalyzeVotes.jsx

```diff
@@ -309,9 +314,10 @@ function AnalyzeVotes() {
 export default AnalyzeVotes;
 
 UserSelector.propTypes = {
-  usernames: PropTypes.arrayOf(PropTypes.string).isRequired,
-  user: PropTypes.string.isRequired,
-  title: PropTypes.string.isRequired,
-  setUser: PropTypes.func.isRequired,
+  arrows: PropTypes.string.isRequired,
```

Comment:
This can be `PropTypes.oneOf(['left-right', 'up-down']), or maybe it should be `navigationDirection: 'horizontal' / 'vertical'`. Either is fine with me, but please update the PropType.

Reply:
I went with `navigationArrows` to be more accurate